app/controllers/staff/funds_controller.rb

```diff
@@ -8,6 +8,9 @@ def create
 
     @activity.wizard_status = "identifier"
     @activity.level = :fund
+    @activity.funding_organisation_name = "HM Treasury"
+    @activity.funding_organisation_reference = "GB-GOV-2"
+    @activity.funding_organisation_type = 10
```

Comment:
We're storing this field as a string but setting an integer. Presumably ActiveRecord is figuring it out for us? It might be clearer to the reader to set this as a string instead.